lib/uvobot/notifications/discourse_notifier.rb

```diff
@@ -46,7 +46,7 @@ def detail_message(detail)
         if detail
           "**Cena:** #{detail[:amount]} EUR  "
         else
-          "**Detaily sa nepodarilo extrahovať.**  "
+          '**Detaily sa nepodarilo extrahovať.**  '
```

Comment:
Este do toho `.join` o poschodie vyssie pridaj aj tie 2 medzery a nemusi to byt tu.